cpu/sensors.py

```diff
@@ -33,6 +33,8 @@ class Sensors(Test):
 
     """
     Run sensors command on ppc architectures.
+
+    :avocado: tags=cpu,power,privileged
```

Comment:
@clebergnu we fixed the doc string can you please have look and possible raise a rectified PR